examples/website/wms/examples.js

```diff
@@ -0,0 +1,35 @@
+export const LOADERS_URI = 'https://raw.githubusercontent.com/visgl/loaders.gl/master';
+
+export const INITIAL_CATEGORY_NAME = 'WMS';
+export const INITIAL_EXAMPLE_NAME = 'Terrestris(OpenStreetMap)';
+
+export const INITIAL_MAP_STYLE =
+  'https://basemaps.cartocdn.com/gl/positron-nolabels-gl-style/style.json';
+
+const VIEW_STATE = {
+  longitude: -122.4,
+  latitude: 37.74,
+  zoom: 9,
+  minZoom: 1,
+  maxZoom: 20,
+  pitch: 0,
+  bearing: 0
+};
+
+export const EXAMPLES = {
+  WMS: {
+    'Terrestris(OpenStreetMap)': {
```

Comment:
I strongly recommend keeping the examples simple and not include too many branches. The hosted version on the website usually have extra features implemented inside `website/src/examples`.